### PR TITLE
Integration test refactor

### PR DESCRIPTION
**Pull request by petebacondarwin**
There are two commits here: 
- [wire up workerd/miniflare logs to Vite](https://github.com/flarelabs-net/vite-plugin-cloudflare/commit/f89b860433b03cdafce66d2aeb581a0ef3de10bf)
- [refactor: make integration tests their own workspace packages](https://github.com/flarelabs-net/vite-plugin-cloudflare/commit/9fde54a134e55c2e1f2d213ed58cfe00f42f7e95)

### CODE REVIEW

**Comment by petebacondarwin** [comment]
This is now ready for a review. The excess SIGTERM handlers I was seeing were because I was creating many vite dev servers in `beforeEach()` handlers but not then cleaning them up.

**Review by jamesopstad — APPROVED** [review]
Looks great! I think we'll need to cleanup the server in the Durable Objects tests too but it doesn't have to be done in this PR.

File: tests/integration/durable-objects/tsconfig.fixtures.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"extends": "@vite-plugin-cloudflare/typescript-config/worker.json",
+	"include": ["**/*.ts"],
+	"exclude": ["*.test.ts"],
+	"compilerOptions": {
+		"noEmit": true
+	}
```
[the remark this answers is not in the document]
Comment (petebacondarwin) [reply tests/integration/durable-objects/tsconfig.fixtures.json:7]: Nice I didn't spot that. I'll remove that.

File: packages/vite-plugin-cloudflare/src/miniflare-options.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { Log, LogLevel } from 'miniflare';
```
[the remark this answers is not in the document]
Comment (petebacondarwin) [reply packages/vite-plugin-cloudflare/src/miniflare-options.ts:1]: I have another PR for reformatting all the imports... see https://github.com/flarelabs-net/vite-plugin-cloudflare/pull/30

**Comment by petebacondarwin** [comment]
> Looks great! I think we'll need to cleanup the server in the Durable Objects tests too but it doesn't have to be done in this PR.

The node-compat PR does some further clean up of tests I think...

File: packages/vite-plugin-cloudflare/src/miniflare-options.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (jamesopstad) [reply packages/vite-plugin-cloudflare/src/miniflare-options.ts:1]: Great! Didn't spot that.

**Pull request merged**